apps/docron/docker-compose.yml

```
@@ -0,0 +1,47 @@
+services:
+  docron:
+    image: ghcr.io/primez/docron:2024.10.13.27
+    container_name: docron
+    ports:
+      - ${APP_PORT}:8080
+    restart: unless-stopped
+    environment:
+     - DOCKER_CONNECTION=unix:///var/run/docker.sock
```

yamllint

[warning] wrong indentation: expected 6 but found 5

(indentation)